seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/utils/JdbcUtils.java

```diff
@@ -0,0 +1,155 @@
+import java.util.Formatter;
+
+public final class JdbcUtils {
+
+    private JdbcUtils() {}
+
+    public static String getString(ResultSet resultSet, int columnIndex) throws SQLException {
+        return resultSet.getString(columnIndex);
+    }
+
+    public static Boolean getBoolean(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getBoolean(columnIndex);
+    }
+
+    public static Byte getByte(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getByte(columnIndex);
+    }
+
+    public static Short getShort(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getShort(columnIndex);
+    }
+
+    public static Integer getInt(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getInt(columnIndex);
+    }
+
+    public static Long getLong(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getLong(columnIndex);
+    }
+
+    public static Float getFloat(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getFloat(columnIndex);
+    }
+
+    public static Double getDouble(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getDouble(columnIndex);
+    }
+
+    public static BigDecimal getBigDecimal(ResultSet resultSet, int columnIndex)
+            throws SQLException {
+        return resultSet.getBigDecimal(columnIndex);
+    }
+
+    public static Date getDate(ResultSet resultSet, int columnIndex) throws SQLException {
+        return resultSet.getDate(columnIndex);
+    }
+
+    public static Time getTime(ResultSet resultSet, int columnIndex) throws SQLException {
+        return resultSet.getTime(columnIndex);
+    }
+
+    public static Timestamp getTimestamp(ResultSet resultSet, int columnIndex) throws SQLException {
+        return resultSet.getTimestamp(columnIndex);
+    }
+
+    public static byte[] getBytes(ResultSet resultSet, int columnIndex) throws SQLException {
+        if (null == resultSet.getObject(columnIndex)) {
+            return null;
+        }
+        return resultSet.getBytes(columnIndex);
+    }
+
+    /** Support to LOG for debug. */
+    public static void formatResultSet(ResultSet resultSet, Appendable appendable)
```

Comment:
What's for?

Reply:
In test print the data. I forgot to delete this code.